src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1249,16 +1249,19 @@ private long EnforceConfigurationFor(DocumentsOperationContext context, string i
                     changeVector, lastModifiedTicks);
 
                 var currentRevisionsCount = GetRevisionsCount(context, id);
+
                 if (needToDeleteMore && currentRevisionsCount > 0)
                     moreWork = true;
 
                 if (currentRevisionsCount == 0)
                 {
+                    var res = _documentsStorage.GetDocumentOrTombstone(context, lowerId, throwOnConflict: false);
```

Comment:
It appears to be a change. Previosuly method could throw in there was a conflict. Do we want that? If yes, do we have a test for that?

Reply:
Actually I'm better without throwing.. it will simply return `null` for both and nothing will happened until the conflict will be resolved.
But yes, having a test to verify the correct behavior here is a good idea. 
Let's do it in the next round of maintenance day?